doc/user/content/serve-results/s3.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Amazon S3"
+description: "How to export results from Materialize to Amazon S3."
+menu:
+  main:
+    parent: sink
+    name: "Amazon S3"
+    weight: 10
+---
+
+{{< private-preview />}}
+
+This guide walks you through the steps required to export results from
+Materialize to Amazon S3. Copying results to S3 is useful to perform tasks like
+periodic backups for auditing, or downstream processing in analytical data
+warehouses like [Snowflake](/serve-results/snowflake/), Databricks or
+BigQuery.
+
+## Before you begin
+
+- Ensure you have access to an AWS account, and permissions to create and manage
+  IAM policies and roles. If you're not an account administrator, you will need
+  support from one!
+
+## Step 1. Set up an Amazon S3 bucket
+
+First, you must set up an S3 bucket and give Materialize enough permissions to
+write files to it. We **strongly** recommend using [role assumption-based authentication](/sql/create-connection/#aws-permissions)
```

Comment:
Because credentials-based authentication has been deprecated. We have a warning in the `CREATE CONNECTION` reference docs: https://materialize.com/docs/sql/create-connection/#credentials-t0

Users _could_ still use it, but I think @benesch is also not a fan of putting it out there as an option too much. I can add a note about credentials-based authentication also being supported and point to the reference documentation. Just wanted to avoid overloading the guides if we don't recommend this approach.

Reply:
Sorry, I meant explain to the user why we recommend this so strongly. Could be as simple as 
> Keeping with AWS security best practices, we **strongly** recommend ...

Reply:
> @benesch is also not a fan of putting it out there as an option too much 

Immaterial clarification: I'm actually a big fan of credentials-based authentication when developing/testing. Unfortunately we're obligated per the terms of our AWS partnership to deprecate credentials-based authentication and push users towards role-based authentication (regardless of whether it's for development only).